interactions/api/http.py

```diff
@@ -357,7 +358,7 @@ async def create_dm(self, recipient_id: int) -> dict:
         # only named recipient_id because of api mirroring
 
         return await self._req.request(
-            Route("POST", "/users/@me/channels"), data=dict(recipient_id=recipient_id)
+            Route("POST", "/users/@me/channels"), json=dict(recipient_id=recipient_id)
```

Comment:
Why are you using a `dict()` literal in this case instead of `{}` dictionary structuring everywhere else in your other class methods? Would help if the codebase is consistent in formatting.

Reply:
Fixed as of c64c4c4.